src/components/IconsContainer/index.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './';
+
+test('renders icons', () => {
+  render(<App />);
+  const appElement = window.document.getElementsByClassName(
+    'IconsContainer'
+  )[0];
```

Comment:
prefer destructuring:
```suggestion
  const [appElement] = window.document.getElementsByClassName(
    'IconsContainer'
  );
```